lm75bd/lm75bd.c

```diff
@@ -9,6 +9,7 @@
 
 /* LM75BD Registers (p.8) */
 #define LM75BD_REG_CONF 0x01U  /* Configuration Register (R/W) */
+#define LM75BD_TEMP_REG 0x00U
```

Comment:
nit: follow the same naming style as the config reg
```suggestion
#define LM75BD_REG_TEMP 0x00U
```